linux_os/guide/services/sssd/sssd-ldap/sssd_ldap_start_tls/rule.yml

```diff
@@ -10,10 +10,10 @@ description: |-
     <br /><br />
     To determine if LDAP is being used for authentication, use the following
     command:
-    <pre>$ sudo grep -i useldapauth /etc/sysconfig/authconfig</pre>
+    <pre>$ sudo systemctl status sssd.service</pre>
     <br /><br />
-    If <tt>USELDAPAUTH=yes</tt>, then LDAP is being used. To check if LDAP is
-    configured to use TLS, use the following command:
+    If the <tt>sssd.service</tt> is <tt>active</tt>, then LDAP is being used.
+    To check if LDAP is configured to use TLS, use the following command:
```

Comment:
Let's fix the rule description here as well, because sssd running doesn't mean that LDAP is being used.

````
    The LDAP client should be configured to implement TLS for the integrity
    of all remote LDAP authentication sessions. If the <tt>id_provider</tt> is
    set to <tt>ldap</tt> or <tt>ipa</tt> in <tt>/etc/sssd/sssd.conf</tt> or any of the
    <tt>/etc/sssd/sssd.conf.d</tt> configuration files, <tt>ldap_id_use_start_tls</tt>
    must be set to <tt>True</tt>.
    <br /><br />
    To check if LDAP is configured to use TLS when <tt>id_provider</tt> is
    set to <tt>ldap</tt> or <tt>ipa</tt>, use the following command:
    <pre>$ sudo grep -i ldap_id_use_start_tls /etc/sssd/sssd.conf</pre>
````

Reply:
Updated in 87c8ac087b45572760833e82f1197f019f7b101d